Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG EXPERIMENTS_VERSION
```

Comment:
Moved from two dockerfiles in `./docker/experiment-lammps(-native).dockerfile` to one single, top-level, `Dockerfile`. I will take this as reference for `faasm/experiment-base#7`.